### PR TITLE
Issue 4322 - Updates old reference to pagure issue

### LINKED ISSUE #4322: Replace Pagure references with GitHub ones
The repo has a lot of references to pagure.io that should be replaced with github.com.
The list includes but is not limited to:

  - man pages;
  - lib389 docs;
  - cockpit-389-ds code comments;
  - core DS code comments;
  - dirsrvtests 'skipif' fixture;

### PR DESCRIPTION
**Pull request by antonionc**
Changes reference from old pagure issue to the corresponding github issue

### CODE REVIEW

**Review by droideck — COMMENTED** [review]
Good catch!
Also, we need to replace all other references to Pagure too.

I'd prefer to do that in this PR and then squash/merge (depending on who's the author) 
I've created an issue for tracking purposes #4322 

Feel free to work on that but I plan to deal with it on this week. :)

File: src/lib389/doc/source/plugin.rst [hunk 1]
```diff
@@ -1,29 +1,29 @@
 Plugin
 ==========
 
-You can take plugin constant names here - https://pagure.io/lib389/blob/master/f/lib389/_constants.py#_164  
+You can take plugin constant names here - https://github.com/droideck/389-ds-base/blob/master/src/lib389/lib389/_constants.py#L179
```
Comment (Firstyear) [inline src/lib389/doc/source/plugin.rst]: This looks like the wrong repository name?

File: src/lib389/doc/source/task.rst [hunk 2]
```diff
@@ -2,31 +2,31 @@ Indexes
 ==========
 
 Besides the predefined tasks (which described in a chapter below) you can create
-your own task objects with specifying a DN (https://pagure.io/lib389/blob/master/f/lib389/_constants.py#_134)
+your own task objects with specifying a DN (https://github.com/droideck/389-ds-base/blob/master/src/lib389/lib389/_constants.py#L146)
```
Comment (Firstyear) [inline src/lib389/doc/source/task.rst]: And here :)

File: src/lib389/doc/source/guidelines.rst [hunk 3]
```diff
@@ -435,7 +425,7 @@ Basic constants
 
 
 For more info check the source code at
-https://pagure.io/lib389/blob/master/f/lib389/_constants.py . If
+https://github.com/droideck/389-ds-base/blob/master/src/lib389/lib389/_constants.py . If
```
Comment (Firstyear) [inline src/lib389/doc/source/guidelines.rst]: One more

**Comment by droideck** [comment]
Fixed!

**Comment by Firstyear** [comment]
Ack, thanks!

**Pull request merged**